### DISCUSSION TITLE
Sofle v2

### DISCUSSION THREAD

**Discussion by alialamri01** [manna-harbour/miryoku#366] [Q&A] [answered]
does anyone have .hex files for the miryoku layout for the sofle keyboard?

**Comment by manna-harbour** [comment, marked as answer]
Sofle is in Miyoku QMK and can be built using the workflows, e.g. https://github.com/manna-harbour/miryoku_qmk/actions/runs/12310586756.

If Sofle v2 is something else it's not in Miryoku QMK or in QMK master, and you'll first have to locate QMK source for it.
